File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/SeatTypeInfo.cs
using System;

namespace ConferenceManagement
{
    public class SeatTypeInfo
    {
        public string Name { get; private set; }
        public string Description { get; private set; }
        public decimal Price { get; private set; }

        public SeatTypeInfo(string name, string description, decimal price)
        {
            Name = name;
            Description = description;
            Price = price;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/models/SeatTypeInfo.java
package com.microsoft.conference.management.domain.models;

import java.math.BigDecimal;


public class SeatTypeInfo {
    private String name;
    private String description;
    private BigDecimal price;

    public SeatTypeInfo() {
    }

    public SeatTypeInfo(String name, String description, BigDecimal price) {
        this.name = name;
        this.description = description;
        this.price = price;
    }

    public String getName() {
        return this.name;
    }

    public String getDescription() {
        return this.description;
    }

    public BigDecimal getPrice() {
        return this.price;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setPrice(BigDecimal price) {
        this.price = price;
    }
}
